packages/aws-cdk-lib/aws-cloudfront-origins/lib/s3-bucket-origin.ts

```diff
@@ -84,21 +84,14 @@ export abstract class S3BucketOrigin extends cloudfront.OriginBase {
         }
 
         if (bucket.encryptionKey) {
-          let bucketName = bucket.bucketName;
-          if (Token.isUnresolved(bucket.bucketName)) {
-            bucketName = JSON.stringify(Tokenization.resolve(bucket.bucketName, {
-              scope,
-              resolver: new DefaultTokenResolver(new StringConcat()),
-            }));
-          }
           Annotations.of(scope).addInfo(
-            `Granting OAC permissions to access KMS key for S3 bucket origin ${bucketName} may cause a circular dependency error when this stack deploys.\n` +
+            `Granting OAC permissions to access KMS key for S3 bucket origin ${bucket.node.id} may cause a circular dependency error when this stack deploys.\n` +
             'The key policy references the distribution\'s id, the distribution references the bucket, and the bucket references the key.\n'+
             'See the "Using OAC for a SSE-KMS encrypted S3 origin" section in the module README for more details.\n',
           );
```

Comment:
Do we still need this info message or can we combine it with the warning about wildcard? Since with the wildcard policy they most likely will not run into the circular dependency error.

Reply:
Suggested a warning message below which combines the two.

Reply:
Agree this message should not be needed anymore. I have removed it in the latest commit. Also updated the other warning to capture the info in your suggest warning message.